Userland/Utilities/grep.cpp

```diff
@@ -168,11 +172,11 @@ int main(int argc, char** argv)
                 return false;
             }
 
-            while (file->can_read_line()) {
+            for (size_t line_number = 0; file->can_read_line(); ++line_number) {
```

Comment:
Shouldn't this start at 1?

Reply:
Argh! Good catch. I was so fixated with the correctness of the *other* loop, that I totally forgot to check this one.